app/src/main/java/com/rob729/quiethours/Database/ProfileRoomDatabase.kt

```diff
@@ -32,42 +32,19 @@ abstract class ProfileRoomDatabase : RoomDatabase() {
                 return instance
             }
         }
-
-        val MIGRATION_1_2: Migration = object : Migration(1, 2) {
-            override fun migrate(database: SupportSQLiteDatabase) {
-                database.execSQL("ALTER TABLE profile_table " +
-                        " ADD COLUMN colorIndex INTEGER NOT NULL DEFAULT 3")
-            }
-        }
-        val MIGRATION_2_3: Migration = object : Migration(2, 3) {
-            override fun migrate(database: SupportSQLiteDatabase) {
-                database.execSQL("ALTER TABLE profile_table " +
-                        " ADD COLUMN vibSwitch INTEGER NOT NULL DEFAULT 1")
-            }
-        }
-        val MIGRATION_3_4: Migration = object : Migration(3, 4) {
-            override fun migrate(database: SupportSQLiteDatabase) {
-                database.execSQL("ALTER TABLE profile_table " +
-                        " ADD COLUMN timeInstance TEXT NOT NULL DEFAULT ''")
-            }
-        }
-        val MIGRATION_4_5: Migration = object : Migration(4, 5) {
-            override fun migrate(database: SupportSQLiteDatabase) {
-                database.execSQL("ALTER TABLE profile_table " +
-                        " ADD COLUMN repeatWeekly INTEGER NOT NULL DEFAULT 0")
-            }
-        }
-        val MIGRATION_5_6: Migration = object : Migration(5, 6) {
-            override fun migrate(database: SupportSQLiteDatabase) {
-                database.execSQL("ALTER TABLE profile_table " +
-                        " ADD COLUMN pauseSwitch INTEGER NOT NULL DEFAULT 1")
-            }
-        }
-        val MIGRATION_6_7: Migration = object : Migration(6, 7) {
-            override fun migrate(database: SupportSQLiteDatabase) {
-                database.execSQL("ALTER TABLE profile_table " +
-                        " ADD COLUMN notes TEXT NOT NULL DEFAULT ''")
+        private fun migrate(a: Int, b: Int, sql: String): Migration {
```

Comment:
Make this private fun migrate(a: Int, b: Int, sql: String) = object : Migration(a, b) {...

Reply:
@rob729 it is not allowing to do so.......giving "Object is not abstract and does not implement abstract base class member"

Reply:
can you once check if removing "object:" works?

Reply:
@rob729 done without removing":object"